packages/design-picker/src/components/pattern-assembler-cta/style.scss

```diff
@@ -15,6 +15,10 @@
 		grid-column: 1 / span 3;
 	}
 
+	@media ( max-width: $break-mobile ) {
+		padding: 32px;
```

Comment:
It might be better to make it as default (or even 16px) and increase the padding via `break-small` and `break-medium`

Reply:
It should be good, let me check

Reply:
Solved